homeassistant/components/saj/sensor.py

```diff
@@ -0,0 +1,161 @@
+"""SAJ solar inverter interface."""
+import asyncio
+from datetime import date, timedelta
+import logging
+
+import pysaj
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_HOST, CONF_SCAN_INTERVAL
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import async_track_time_interval
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSOR_ICON = "mdi:solar-power"
+
+
+PLATFORM_SCHEMA = vol.All(
+    PLATFORM_SCHEMA.extend(
+        {vol.Required(CONF_HOST): cv.string}, extra=vol.PREVENT_EXTRA
+    )
+)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up SAJ sensors."""
+
+    # Init all sensors
+    sensor_def = pysaj.Sensors()
+
+    # Use all sensors by default
+    hass_sensors = []
+    used_sensors = []
+
+    for sensor in sensor_def:
+        hass_sensors.append(SAJsensor(sensor))
+        used_sensors.append(sensor)
```

Comment:
Do we need to create another list for the saj sensors? Can we use `sensor_def` directly instead?

Reply:
The `used_sensors` is indeed unnecessary, thanks for checking, I'll change it.